hibernate-core/src/main/java/org/hibernate/query/spi/QueryEngine.java

```diff
@@ -281,7 +290,18 @@ private static SqmTranslatorFactory resolveSqmTranslatorFactory(
 		return new StandardSqmTranslatorFactory();
 	}
 
-	private static QueryInterpretationCache buildInterpretationCache(Supplier<StatisticsImplementor> statisticsSupplier, Map properties) {
+	private static List<FunctionContributor> sortedFunctionContributors(ServiceRegistry serviceRegistry) {
```

Comment:
I'm missing where this is used

Reply:
Ok, I see it now.  I still think adding it to the `#contributeFunctions` method is better.

However, I wonder why the "position" at all.   Such things tend to be more difficult than useful.

Reply:
To avoid possible difficulties with ordering when we run on the class path :)

Reply:
I don't think it solves any difficulties